mozfldp/client.py

```diff
@@ -69,8 +71,20 @@ def update_and_submit_weights(
                     self._features[batch_ind], self._labels[batch_ind]
                 )
 
-        # TODO: submit new weights to the server via API request.
-        # self._model.get_weights() -> server
+        # load the client weight into json payload
+        client_data = {
+            "coefs": current_coef.tolist(),
+            "intercept": current_intercept.tolist(),
+            "num_samples": self._n
+        }
+
+        payload = json.dumps(client_data)
+
+        # send the post request to update the weights
+        api_endpoint = "http://0.0.0.0:8000/api/v1/ingest_client_data/{}".format(self._id)
```

Comment:
@crankycoder The URL probably shouldn't be hardcoded, but what do you think is the best way to handle it - pass in on instantiation?

Reply:
Let's use https://pypi.org/project/python-decouple/ to pull hostname and port from enviroment variables and use defaults of `127.0.0.1` for host and `8000` for port.

The code will end up looking something like:
```
from decouple import config

HOSTNAME = config('FLDP_HOST', '127.0.0.1')
PORT = config('FLDP_PORT', 8000)

...

api_endpoint = "http://{hostname:s}:{port:d}/api/v1/ingest_client_data/{id:d}".format(hostname=HOSTNAME, port=PORT, id=self._id) 

```